config/utils.js

```diff
@@ -100,64 +100,37 @@ function fileHashsum(filePath) {
 }
 exports.fileHashsum = fileHashsum;
 
-exports.getBuildInfo = function (packageJson, sqExtensionManifest, scExtensionManifest) {
-  const sqPackageVersion = getVersionWithCirrusBuildNumber(sqExtensionManifest.version);
-  const sqVsixPaths = globby.sync(path.join(DIST_DIR, `*-sonarqube.vsix`));
-  const sqAdditionalPaths = globby.sync(
-    path.join(DIST_DIR, `*{cyclonedx-sonarqube-*.json,cyclonedx-latest.json,-sonarqube*.asc}`),
+exports.getBuildInfo = function (type, vssData) {
+  const typeAccronym = type === "sonarqube" ? "sq" : "sc";
```

Comment:
NP: the word "type" is too vague imo 🤷🏻 
```suggestion
  const productAccronym = type === "sonarqube" ? "sq" : "sc";
```